main.go

```diff
@@ -122,9 +123,25 @@ func main() {
 
 	ctrl.SetLogger(zap.Logger(true))
 
+	err := resourcemanagerconfig.ParseEnvironment()
+	if err != nil {
+		setupLog.Error(err, "unable to parse settings required to provision resources in Azure")
+		os.Exit(1)
+	}
+
+	setupLog.V(0).Info("Configuration details", "Configuration", resourcemanagerconfig.ConfigString())
+
+	targetNamespaces := resourcemanagerconfig.TargetNamespaces()
+	var cacheFunc cache.NewCacheFunc
+	if targetNamespaces != nil {
+		setupLog.V(0).Info("Restricting operator cache", "namespaces", targetNamespaces)
```

Comment:
minor: This is technically pretty much duplicate as it gets logged by `ConfigString()` anyway?

It might be worth keeping just because it adds some nice explicitness/clarity for customers looking at logs, but I don't see any extra value we'd be gaining from it over and above the log above on line 132

Reply:
Yeah, I think that was really just for me while I was implementing it - removed!